lib/pf/provisioner/mobileconfig.pm

```diff
@@ -25,29 +25,71 @@ The set the default OS to IOS
 =cut
 
 # Will always ignore the oses parameter provided and use ['Apple iPod, iPhone or iPad']
-has 'oses' => (is => 'ro', default => sub { ['Apple iPod, iPhone or iPad'] }, coerce => sub { ['Apple iPod, iPhone or iPad'] });
+has 'oses' => (is => 'ro', default => sub { ['Apple iPod, iPhone or iPad', 'Mac OS X Lion'] }, coerce => sub { ['Apple iPod, iPhone or iPad', 'Mac OS X Lion'] });
 
 =head2 ssid
 
-The ssid
+The ssid boarcast name
```

Comment:
boarcast ? What does a boar has to do with this ?

![image](https://cloud.githubusercontent.com/assets/3857942/7434758/59e33a7e-f00b-11e4-80e4-4949a3b0d1f7.png)